tests/e2e/upgrade/node.go

```diff
@@ -0,0 +1,41 @@
+package upgrade
+
+import "github.com/ory/dockertest/v3"
+
+var baseCmd = []string{"evmosd", "start"}
+
+type Node struct {
```

Comment:
can you add comments on each public struct/func?

Reply:
@fedekunze added in places where name is not self-descriptive